card/db/migrate_core_cards/data/responsive_sidebar.json

```diff
@@ -0,0 +1,26 @@
+{
+  "url": "http://localhost:3000/export.json",
+  "timestamp": "2015-05-01 03:06:37 +0200",
+  "card": {
+    "name": "export",
+    "type": "Pointer",
+    "content": "[[style: right sidebar]]\n[[Default Layout]]\n[[*header]]",
+    "value": [
+      {
+        "name": "style: right sidebar",
+        "type": "SCSS",
+        "content": "/*\r\nThis stylesheet defines a simple layout with the sidebar at right.\r\n*/\r\n#primary {\r\n  margin: 0 1% 3em 1%;\r\n  float: left;\r\n  width: 67%;\r\n}\r\n\r\n#secondary {\r\n  margin: 0 1% 3em 1%;\r\n  float: right;\r\n  width: 28%;\r\n}\r\n\r\nfooter {\r\n  clear:both;\r\n  text-align: center;\r\n  padding: 1em;\r\n}\r\n\r\n@media print {\r\n  article,\r\n  #primary {\r\n    width: 100%;\r\n    float: none;\r\n    margin: 0;\r\n  }\r\n\r\n  aside,\r\n  #secondary,\r\n  footer {\r\n    display: none;\r\n  }\r\n}\r\n"
```

Comment:
I removed `article` and  `aside` here. This change will make decks with the old layout look ugly but it is necessary to use the bootstrap grid.